File: api-signature/src/tr/gov/tubitak/uekae/esya/api/signature/certval/ReferencedOCSPResponseFinder.java
package tr.gov.tubitak.uekae.esya.api.signature.certval;

import tr.gov.tubitak.uekae.esya.api.asn.ocsp.EOCSPResponse;
import tr.gov.tubitak.uekae.esya.api.common.ESYAException;

import java.util.List;

/**
 * @author ayetgin
 */
public interface ReferencedOCSPResponseFinder
{

    List<EOCSPResponse> find(OCSPSearchCriteria aCriteria) throws ESYAException;

}

File: api-signature/csharp/src/certval/ReferencedOCSPResponseFinder.cs
using System.Collections.Generic;
using tr.gov.tubitak.uekae.esya.api.asn.ocsp;

namespace tr.gov.tubitak.uekae.esya.api.signature.certval
{
    public interface ReferencedOCSPResponseFinder
    {
        List<EOCSPResponse> find(OCSPSearchCriteria aCriteria);
    }
}
